mllib/src/main/scala/org/apache/spark/mllib/stat/MultivariateOnlineSummarizer.scala

```diff
@@ -47,6 +49,39 @@ class MultivariateOnlineSummarizer extends MultivariateStatisticalSummary with S
   private var currMax: Array[Double] = _
   private var currMin: Array[Double] = _
 
+  private var configure: mutable.Map[String, Boolean] = mutable.Map("mean" -> false, "max" -> false,
+    "min" -> false, "variance" -> false, "normL1" -> false, "normL2" -> false) 
+
+  def withMean(flag: Boolean): this.type = {
+    configure("mean") = flag
+    this
+  }
+
```

Comment:
okay. I see why you want to use `map`. it will be easier to compare with the other map. since those are private, i think it's okay for now. please have local copy as `val` in function as the following.

also, the variable name in function will be part of api. please have it more descriptive like

```scala
 def setWithMax(withMax: Boolean): this.type = {
   configure("withMax") = withMax
   this
 }

```